types/src/block.rs

```diff
@@ -3,54 +3,57 @@ mod block_hash;
 mod block_hash_and_height;
 mod block_header;
 mod block_signatures;
+mod block_v1;
+mod block_v2;
 mod era_end;
 mod era_report;
 mod finality_signature;
 mod finality_signature_id;
 mod json_compatibility;
 mod signed_block_header;
+#[cfg(any(feature = "testing", test))]
+pub mod test_block_builder;
 
-#[cfg(any(all(feature = "std", feature = "testing"), test))]
-use alloc::collections::BTreeMap;
 use alloc::{boxed::Box, vec::Vec};
 use core::fmt::{self, Display, Formatter};
-#[cfg(any(all(feature = "std", feature = "testing"), test))]
-use core::iter;
+#[cfg(feature = "once_cell")]
```

Comment:
Shouldn't this be `#[cfg(feature = "json-schema")]` instead since we use `Lazy` only for defining `static BLOCK` below?

Reply:
Yes, good catch. Fixed in 60d1b481e6b1ea24a92343c49ce6b724cb15f6b9 and 06fff783c44ae7a795387e3cc60c63bee9784c93 along with the update to `json_block_with_signatures.rs`.